apps/chat/src/store/conversations/conversations.epics.ts

```diff
@@ -234,90 +252,62 @@ const initSelectedConversationsEpic: AppEpic = (action$, state$) =>
-            };
-          }
-
-          return {
-            conversations: selectedConversations,
-            selectedConversationsIds: selectedIds.filter((id) =>
-              selectedConversations.some((conv) => conv.id === id),
-            ),
-          };
-        }),
-        switchMap(({ conversations, selectedConversationsIds }) => {
-          const isIsolatedView = SettingsSelectors.selectIsIsolatedView(
-            state$.value,
-          );
-
-          // Always create new conversation in isolated view
-          if (isIsolatedView) {
-            const isolatedModelId = SettingsSelectors.selectIsolatedModelId(
-              state$.value,
-            );
 
+        switchMap(({ selectedConversations, selectedIds }) => {
+          if (!selectedConversations.length) {
             return of(
               ConversationsActions.createNewConversations({
-                names: [`isolated_${isolatedModelId}`],
-                shouldUploadConversationsForCompare: true,
+                names: [translate(DEFAULT_CONVERSATION_NAME)],
               }),
             );
           }
 
-          const actions: Observable<AnyAction>[] = [
-            of(
-              ConversationsActions.selectConversations({
-                conversationIds: selectedConversationsIds,
-              }),
-            ),
-          ];
+          const selectedConversationsIds = selectedIds.filter((id) =>
+            selectedConversations.some((conv) => conv.id === id),
+          );
 
-          if (conversations.length) {
-            return concat(
-              of(
-                ConversationsActions.addConversations({
-                  conversations: conversations.map((conv) => {
-                    if (!isEntityIdPublic(conv)) {
-                      return conv;
-                    }
-
-                    const parsedApiKey = parseConversationApiKey(
-                      splitEntityId(conv.id).name,
-                      { parseVersion: true },
-                    );
+          const actions: Observable<AnyAction>[] = [];
 
-                    return {
-                      ...conv,
-                      name: parsedApiKey.name,
-                      publicationInfo: parsedApiKey.publicationInfo,
-                    };
-                  }),
-                }),
-              ),
+          if (selectedConversationsIds) {
+            actions.push(
               of(
-                UIActions.setOpenedFoldersIds({
-                  openedFolderIds: selectedConversationsIds.flatMap(
-                    getParentFolderIdsFromEntityId,
-                  ),
-                  featureType: FeatureType.Chat,
+                ConversationsActions.selectConversations({
+                  conversationIds: selectedConversationsIds,
                 }),
               ),
-              ...actions,
             );
           }
 
           return concat(
-            ...actions,
             of(
```

Comment:
why did you remove `createNewConversations` action?

Reply:
It was not removed. This logic has been moved up to the line 258.